ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/WriteConfigService.scala

```diff
@@ -0,0 +1,29 @@
+package com.daml.ledger.participant.state.v1
+
+import java.util.concurrent.CompletionStage
+
+import com.digitalasset.daml.lf.data.Time.Timestamp
+
+trait WriteConfigService {
```

Comment:
Is this interface mandatory for all X's? I am thinking especially here about Daml-on-SQL and Canton.

Reply:
they can choose to return "NotImplemented" error.

Reply:
We should write that in the comment then

Reply:
commented.